libraries/Microsoft.Bot.Schema/Converters/SurfaceConverter.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Schema.Converters
```

Comment:
Does this converter actually get used anywhere in the SDK? This converter only has support to serialize from JSON to the meeting notification models. This isn't super helpful from a bot perspective since they are the ones sending the JSON to the server. Unless the intention of including this is to help developers directly serialize raw JSON strings they might have defined

Reply:
You are right Vu. The intention of this SurfaceConverter is to help developer deserialize JSON string to object.

Reply:
Per offline discussion, I'm good with keeping this